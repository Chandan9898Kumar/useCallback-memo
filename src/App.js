import { BrowserRouter, Route, Routes } from "react-router-dom";
import './App.css';
import Home from './Home';
import Parents from './UseCallBack/Parent';
import MemoComponent from './Memo/MemoComponent'
function App() {
  return (
    <div className="App">
    <label>My React App</label>
      <BrowserRouter>
        <Routes>
          <Route exact path='/' element={<Home />} />
          <Route exact path='/useCallback' element={<Parents />} />
          <Route exact path='/memoCom' element={<MemoComponent />} />

        </Routes>
      </BrowserRouter>
    </div>
  );
}

export default App;
