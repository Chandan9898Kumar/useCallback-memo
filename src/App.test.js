import { render, screen } from '@testing-library/react';
import App from './App';

test('renders learn react link', () => {
  render(<App />);
  const linkElement = screen.getByText(/My React App/);
  expect(linkElement).toBeInTheDocument();
  expect(linkElement).toBeDefined()
});

test('It should be defined', () => {
  render(<App />);
  const linkElement = screen.getByText(/My React App/);
  expect(linkElement).toBeDefined()
});

