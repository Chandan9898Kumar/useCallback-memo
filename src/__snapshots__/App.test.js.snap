// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`To Match the Snapshot 1`] = `
<label>
  My React App
</label>
`;
